implementation/tracing/src/connector_impl.cpp

```diff
@@ -269,7 +247,8 @@ void connector_impl::trace(const byte_t *_header, uint16_t _header_size,
                     // This should never happen!
                     VSOMEIP_ERROR << "tracing: found channel without DLT context!";
                 }
-            #else
+            #endif
+            #ifdef TRACE_TO_LOGS
```

Comment:
last change
```suggestion
            #else

Reply:
this way if you have USE_DLT and TRACE_TO_LOGS defined, you don't print the same message twice

Reply:
No worries, I’m happy to make as many changes as needed to align this PR with the project’s standards. This was a good catch, thank you for pointing it out!